Packs/CortexXDR/Integrations/CortexXDRIR/README.md

```diff
@@ -2799,7 +2799,8 @@ BLOCKED_TRIGGER_4: prevented \(on write\)
 
 ### xdr-get-contributing-event
 ***
-Retrieves contributing events for a specific alert.
+Retrieves contributing events for a specific correlation alert.
+Known limitation: the command is compatible **only** with correlation alerts, otherwise an error will be raised.
```

Comment:
Was this added by you? or some info from updated yml?

Reply:
Was added by me, found that out when I tried to run this command, after consulting @GuyAfik I learned that only specific kind of alerts is cooperating with the command so I documented that.